### PR TITLE
Fix Accumulate Module

### PR DESCRIPTION
**Pull request by trisyoungs**
This PR fixes and updates the `Accumulate` module to properly accumulate separate module targets, as well as providing a mechanism to display the results in the GUI.

### CODE REVIEW

File: src/modules/accumulate/gui/accumulateWidgetFuncs.cpp [hunk 1]
```diff
@@ -141,3 +172,22 @@ void AccumulateModuleWidget::on_PartialsButton_clicked(bool checked)
...
+    currentTargetModule_ = ui_.ModuleTargetCombo->currentData().value<Module *>();
+
+    updateControls(ModuleWidget::RecreateRenderablesFlag);
+}
+
+void AccumulateModuleWidget::on_FilterEdit_textChanged(QString text) { updateControls(ModuleWidget::RecreateRenderablesFlag); }
+
+void AccumulateModuleWidget::on_ClearFilterButton_clicked(bool checked)
+{
+    ui_.FilterEdit->setText("");
```
Comment (jws-1) [inline src/modules/accumulate/gui/accumulateWidgetFuncs.cpp:190]:
```suggestion
    ui_.FilterEdit->clear();
```

File: src/modules/accumulate/process.cpp [hunk 2]
```diff
@@ -70,8 +70,8 @@ bool AccumulateModule::process(Dissolve &dissolve, const ProcessPool &procPool)
         // Save data if requested
         std::vector<std::string> suffixes = {"gr", "sq", "gr"};
         std::vector<std::string> units = {"r, Angstroms", "Q, Angstroms**-1", "r, Angstroms"};
-        if (save_ &&
-            (!MPIRunMaster(procPool, accumulated.save(name_, dataName, suffixes[targetPartialSet_], units[targetPartialSet_]))))
+        if (save_ && (!MPIRunMaster(procPool, accumulated.save(fmt::format("{}-{}", name(), targetModule->name()), dataName,
```
Comment (jws-1) [inline src/modules/accumulate/process.cpp:73]: Possibly more readable?
```suggestion
        if (save_ && !(MPIRunMaster(procPool, accumulated.save(fmt::format("{}-{}", name(), targetModule->name()), dataName,
```

**Review by jws-1 — APPROVED** [review]
Couple of minor comments, but looks good otherwise.

File: src/modules/accumulate/process.cpp (hunk 2 above)
Comment (trisyoungs) [reply src/modules/accumulate/process.cpp:73]: Yeah, probably!  MPI still exists in the code for now....

**Pull request merged**